packages/app/utils/errors.ts

```diff
@@ -3,7 +3,7 @@ import type { NextApiResponse } from 'next'
 import type { ZodError, ZodSchema } from 'zod'
 import { parameterWithInflection } from '../lib/grammar'
 
-export enum ErrorCode {
+export enum ErrorStatusText {
```

Comment:
This is it—the once change that touched a million files. I'm proposing that we call this `ErrorStatusText` because it represents the [response.statusText](https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText) interface. The progenitor of this change was modifying (in another PR) `HttpException` to accept a `response.status` so that we could generate a compatible custom error from another API's response (when `!response.ok`).